packages/cli/src/Constants.ts

```diff
@@ -93,6 +93,87 @@ For ${Constants.DISPLAY_NAME} support, visit ${Constants.SUPPORT_LINK}
      */
     public static readonly HOME_DIR = "~/.zowe";
 
+    public static readonly AUTO_INIT_OPTION_GROUP = "APIML Connection Options";
+
+    /**
+     * Option used in profile creation and commands for hostname
+     */
+    public static AUTO_INIT_OPTION_HOST: ICommandOptionDefinition = {
```

Comment:
There seems to be a similar set of options (`BASE_OPTION_HOST`), should they be shared and the group changed on the handler definition?
Not sure if we will ever change these options description or aliases, but if we did... we will have to change them in more than one place.

Reply:
There are some differences, mainly the group and descriptions. I'm not sure if I follow what you mean about changing the group on the handler definition. Would it be okay if we use the spread operator on the existing options when defining the new ones, and just override the group and description?

Reply:
Sorry, handler and definition are two separate things. 
I meant the command definition since these options eventually make their way into a `.definition.ts` file.
As you said, the group and description could be replaced at that time : )
IDK if this needs done as part of this PR though. We can worry about that at a later time 😋